### ISSUE TITLE
Install of CLI tool fails with 500 error on postgres dependency

### ISSUE THREAD

**Issue by michaelachrisco** [denolib/typeorm#105]
**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Deno version:**

`x.x.x`

**Steps to reproduce or a small repository showing the problem:**
```
deno install --allow-read --allow-write --allow-net --allow-env -f -n typeorm https://deno.land/x/typeorm/cli.ts
Download https://deno.land/x/typeorm/cli.ts
...
Download https://deno.land/x/mysql@v2.3.0/src/constant/charset.ts
Download https://raw.githubusercontent.com/chiefbiiko/std-encoding/v1.0.0/mod.ts
error: Import 'https://deno.land/x/postgres@28249fbec10f7f1581f405a5a5b4b105a838629b/mod.ts' failed: 404 Not Found
Imported from "https://deno.land/x/typeorm@v0.2.23-rc6/vendor/https/deno.land/x/postgres/mod.ts:1"
```
First time run of clit install is failing.
It appears as though https://deno.land/x/postgres@28249fbec10f7f1581f405a5a5b4b105a838629b/mod.ts' is failing with a 500 error. Should the driver reference be updated?

**Comment by michaelachrisco** [comment]
Thanks @uki00a !

**Comment by uki00a** [comment]
@michaelachrisco 
This problem has been fixed at [v0.2.23-rc7](https://github.com/denolib/typeorm/releases/tag/v0.2.23-rc7)!
Thnaks for your reporting!
